extensions/prototypes/labgraph_monitor/.husky/pre-commit

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
```

Comment:
Is .husky file a compulsory file for this commit?

Reply:
Currently I'm using [prettier](https://prettier.io/docs/en/index.html), [eslint](https://eslint.org/docs/user-guide/getting-started), [husky]() to automate formatting, identifying patterns which keeps the code consistent and production-ready.
I believe that aligns with the approach that LabGraph is using with python by using flake8. The point of using husky is to automate the process through git-hooks, my current code will run a check whenever a contributor tries to add a new commit(pre-commit git-hook) that will prevent bad commits if there are any (e.g: unwanted console.log(), unused packages, unformatted code ...).
The husky script will run only if there is a change in the **extensions/prototypes/labgraph_monitor** directory.